packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-accessors/undecorated-static-private/output.js

```diff
@@ -15,3 +15,19 @@ class Foo {
     Foo.#B = v;
   }
 }
+Foo = class _Class {
```

Comment:
> The named evaluation is still preserved because Babel generates specific setFunctionName calls for those will-be-transformed classes.

It doesn't seem to be the case here?

Reply:
Fixed in https://github.com/babel/babel/pull/16485/commits/f45d9b31a94df070bdc9c816fa5a41db73d35dcf.